content/blog/2022/02/multiplatform-kubewarden.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Multiplatform Kubewarden
+authors:
+    - Rafael Fernández López
+date: 2022-02-02
+---
+
+The Kubewarden team is glad to announce that in the spirit of helping
+Policy Authors and Cluster Administrators, the project is now
+officially multiplatform.
+
+The list of supported platforms as of now are:
+
+- Policy Server, as a [container image](https://github.com/kubewarden/policy-server/pkgs/container/policy-server):
+  - `linux/amd64` (with the `musl` libc)
+  - `linux/arm64` (with the `musl` libc)
+
+- kwctl, as a [standalone binary](https://github.com/kubewarden/kwctl/releases):
+  - `darwin` (`x86_64`)
+  - `linux` (`aarch64`, with the `musl` libc)
+  - `linux` (`x86_64`, with the `musl` libc)
+  - `windows` (`x86_64`, with `MSVC`)
+
+We have prioritized the usage of the same dependencies and toolchain
+on platforms where we were able to do so.
+
+## Looking for platform convergence
+
+Initially, we were using `glibc` and `openssl` as dependencies in our
+project. However, these two libraries are somewhat special and
+sometimes problematic to converge into other platforms.
+
+Here is a short resume of the lessons learned, sometimes the hard way,
+for both libraries.
+
+### The libc
+
+In the early stage of development, the Kubewarden team decided to use the [`musl libc`](https://musl.libc.org/) library.
+This allowed us to avoid potential difficulties with `glibc` down the road (i.e. Cross-compilation time or making a full static binary).
+
+The team also thinks that producing a final static binary for all platforms is in our users own interests; specially for a CLI tool like `kwctl`, which interacts directly with our users' environment.
+
+### The SSL stack
+
+The second part of the puzzle is `openssl`. It's also a complex piece
+of software.
+
+As stated above, we are interested in cross compiling for multiple
+Operating Systems such as Windows and MacOS (Darwin).
+
+Lucky us, a new TLS implementation has popped up in the Rust
+ecosystem: [rustls](https://github.com/rustls/rustls). Thanks to
+`rustls` it's now easier to produce a final binary of `kwctl` that
+will work on all major platforms. And all that, from the same source
+code.
+
+## Closing
+
+This decisions might seem very technical or bringing "not-so-high"
+value as other features. But for the Kubewarden team, it's important
```

Comment:
```suggestion
value compared to other features. But for the Kubewarden team, it's important
```

Reply:
Wouldn't as compared to other features be more appropriate?

Reply:
This is a non-blocking stylistic nit that improves readability, IMO. We can modify this later on, if need be.